PeerioTechnologies/peerio-mobile
# Chore react native 0.55

#### Relevant info and issue/PR links 

https://app.clubhouse.io/peerio/story/13357/chore-upgrade-to-latest-stable-react-native
 
#### Testing instructions  

Clone the repo to a separate folder and switch to a branch.
Binary requires full node_modules reinstall and rebuild.

Things to take notice for:

- FS interactions
- network interactions
- animations

Known issues:

- no layout animation working on android, most animations are disabled. the fix is on the way
https://app.clubhouse.io/peerio/story/14097/mobile-android-fix-layoutanimation-on-transitions-for-rn-0-55

- chat list item height calculations on android are broken, no calculations applied right now, which may cause incorrect "unread chat indicator" positioning. here's the story:

https://app.clubhouse.io/peerio/story/14098/mobile-android-fix-height-calculation-for-chat-list
